vmm/src/vm_config.rs

```diff
@@ -495,6 +495,8 @@ pub struct VsockConfig {
     pub id: Option<String>,
     #[serde(default)]
     pub pci_segment: u16,
+    #[serde(default)]
+    pub host_listen: bool,
```

Comment:
I would prefer this to be `mode` like `VhostMode`.

Reply:
> I would prefer this to be `mode` like `VhostMode`.

Thx for the suggestions.

I suppose @weltling 's proposal which make `socket=` optional is a better way? If we do this way, this `host_listen` can be removed :)

Reply:
Not passing `socket=` would just mean the communication `host->guest` is not desired. It would still allow having `guest->host`. But yep, having `mode` explicitly with some keywords like `mode=bidirectional`, `mode=host-to-guest`, `mode=guest-to-host` would be a more verbose option. For the current PR it wolud probably mean more effort, though.

Thanks

Reply:
Oh yes, in general people specify the unix socket in host according to the parameter in cloud hypervisor no matter guest->host or host->guest. If we made socket optional, that may make people confused, I suppose 😺